plugins/catalog/src/components/EntityLayout/EntityLayout.tsx

```diff
@@ -68,12 +68,21 @@ const EntityLayoutTitle = ({
 }: {
   title: string;
   entity: Entity | undefined;
-}) => (
-  <Box display="inline-flex" alignItems="center" height="1em">
-    {title}
-    {entity && <FavouriteEntity entity={entity} />}
-  </Box>
-);
+}) => {
+  return (
+    <Box display="inline-flex" alignItems="center" height="1em" maxWidth="100%">
```

Comment:
I guess we _are_ using `em`s but only _sometimes_?

Reply:
Yes - this is a good question & was wondering as well.

The underlying problem when tackling this might be, that there is not a proposed way of MUI to handle this kind of stuff in general. There is an [ongoing discussion on using rem/em for components, when no absolute pixel value is needed](https://github.com/material-components/material-components-web/issues/2731) suggesting that this might change/be tackled in v5. Even the [font scaling does not work with rem/em yet leading to a11y problems](https://github.com/material-components/material-components-web-components/issues/1751).

That said - there are helper functions converting `px` to `rem` allowing overwriting of e.g. the `font-size` in the theme:
![Screenshot 2021-07-16 at 17 12 50](https://user-images.githubusercontent.com/8904624/125971689-537d5a0c-27d3-493b-8171-09c5ac1382e8.png)

I just got an overview of this topic & it might be nice to discuss this. Keeping it to everyone to decide when to use px/rem/em & wether static or from the theme seems definitely kind of wrong.